include/z64scene.h

```diff
@@ -517,8 +514,8 @@ typedef enum {
 #define SCENE_CMD_ECHO_SETTINGS(echo) \
     { SCENE_CMD_ID_ECHO_SETTINGS, 0, CMD_BBBB(0, 0, 0, echo) }
 
-#define SCENE_CMD_CUTSCENE_LIST(numCutscene, cutsceneList) \
-    { SCENE_CMD_ID_CUTSCENE_LIST, numCutscene, CMD_PTR(cutsceneList) }
+#define SCENE_CMD_DEFAULT_CUTSCENE(defaultCutscene) \
+    { SCENE_CMD_ID_CUTSCENE_LIST, 0, CMD_PTR(defaultCutscene) }
```

Comment:
Not sure if I'd call this "default" cutscene

Reply:
There is only one scripted cutscene per scene setup in oot so default should be omitted, as there is only one.

Reply:
So `cutsceneData` should be fine?

Reply:
yes, and the command name should reflect it as well